requirements.txt

```diff
@@ -15,3 +15,6 @@ pycrypto==2.6.1
 requests==2.4.3
 six==1.7.3
 wsgiref==0.1.2
+django-easy-pdf==0.1.0
```

Comment:
It's better to dump the requirements into the requirements file than modify them directly, because the requirements may have other dependencies. You can do this with
    pip freeze > requirements.txt

Obviously, you should make sure you're in the virtualenv and you don't have any extra packages installed in the virtualenv, otherwise that'll be listed in the requirements file also.